jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/card/CardKitchenSinkTests.java

```diff
@@ -0,0 +1,116 @@
+    private WebElement link0;
+    private WebElement link1;
+    private ArrayList<String> tabs;
+
+    @BeforeMethod
+    public void before() {
+        shouldBeLoggedIn();
+        bsPage.shouldBeOpened();
+    }
+
+    @Test
+    public void itemsTest() {
+        assertEquals(cardKitchenSink.list.size(), 3);
+        cardKitchenSink.list.get(0).is().text(is(item0Text));
+        cardKitchenSink.list.get(1).is().text(is(item1Text));
+        cardKitchenSink.list.get(2).is().text(is(item2Text));
+        cardKitchenSink.list.get(0).is().text(containsString(item0Text));
+        cardKitchenSink.list.get(1).is().text(containsString(item1Text));
+        cardKitchenSink.list.get(2).is().text(containsString(item2Text));
+    }
+
+    @Test
+    public void baseValidationTest() {
+        baseValidation(cardKitchenSink.image);
+        baseValidation(cardKitchenSink.title);
+        baseValidation(cardKitchenSink.text);
+        baseValidation(cardKitchenSink.list);
+        baseValidation(cardKitchenSink.list.get(0));
+        baseValidation(cardKitchenSink.list.get(1));
+        baseValidation(cardKitchenSink.list.get(2));
+        baseValidation(cardKitchenSink.body.get(0));
+        baseValidation(cardKitchenSink.body.get(1));
+    }
+
+    @Test
+    public void isValidationTest() {
+        cardKitchenSink.image.is().src(is(imgSrc));
+        cardKitchenSink.image.is().alt(is(imgAlt));
+        cardKitchenSink.image.unhighlight();
+        cardKitchenSink.image.assertThat().width(is(86));
+        cardKitchenSink.image.assertThat().height(is(137));
+        cardKitchenSink.title.is().text(is(titleText));
+        cardKitchenSink.title.is().text(containsString(titleText));
+        cardKitchenSink.text.is().text(is(cardText));
+        cardKitchenSink.text.is().text(containsString(cardText));
+    }
+
+    @Test
+    public void linksTest(){
+        link0 = cardKitchenSink.body.get(1).findElements(By.cssSelector(".card-link")).get(0);
+        assertTrue(link0.isDisplayed());
+        assertEquals(link0.getText(), link0Text);
+        assertEquals(link0.getAttribute("target"), "_blank");
+        assertEquals(link0.getTagName(), "a");
+
+        link1 = cardKitchenSink.body.get(1).findElements(By.cssSelector(".card-link")).get(1);
+        assertTrue(link1.isDisplayed());
+        assertEquals(link1.getText(), link1Text);
+        assertEquals(link1.getAttribute("target"), "_blank");
+        assertEquals(link1.getTagName(), "a");
+
+        WebDriver driver = WebDriverFactory.getDriver();
+
+        link0.click();
+        refreshTabs();
+        driver.switchTo().window(tabs.get(1));
+        assertEquals(getUrl(), link0Url);
+        driver.close();
+        driver.switchTo().window(tabs.get(0));
+
+        link1.click();
+        refreshTabs();
+        driver.switchTo().window(tabs.get(1));
+        assertEquals(getUrl(), link1Url);
+        driver.close();
+        driver.switchTo().window(tabs.get(0));
```

Comment:
switchTo() after driver.close().. is it planned?

Reply:
Yes, because we close just a tab